### PR TITLE
Add link about how to provide an endpoint to README.md

### PR DESCRIPTION
**Pull request by zeno4ever**
added some info to hopefully prevent commen mistakes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,8 @@ If your space is missing in the [list](./directory.json), [fork the repository](
 add your space and create a pull request in GitHub.
 
 Find more information about the SpaceAPI on [spaceapi.io](https://spaceapi.io).
+
+Some tips for correct implemetation :
+* Check if your json file is valid with online [validator](https://spaceapi.io/validator/).
+* Tripple check your longatude/latatude if they are not swapped.
+* Check if CORS is enabled (adding a Access-Control-Allow-Origin: * header to the webserver).
```
Comment (rnestler) [inline README.md:15]: Maybe we could just point at https://spaceapi.io/provide-an-endpoint/ instead?

Comment (zeno4ever) [reply README.md:15]: Sure, we can do that if you prefer that. In my experience people read documentation poorly (and I saw in the pull request many times the same mistakes) so repeating a bit of documentation can't harm IMHO.

File: README.md [hunk 2]
```diff
@@ -8,3 +8,5 @@ If your space is missing in the [list](./directory.json), [fork the repository](
 add your space and create a pull request in GitHub.
 
 Find more information about the SpaceAPI on [spaceapi.io](https://spaceapi.io).
+
+Some tips for correct implemetation see information about [providing a endpoint](https://spaceapi.io/provide-an-endpoint/).
```
Comment (rnestler) [inline README.md:12]:
```suggestion
Some tips for correct implementation see information about [providing a endpoint](https://spaceapi.io/provide-an-endpoint/).
```

**Review by rnestler — APPROVED** [review]

**Comment by rnestler** [comment]
Thanks @zeno4ever

**Pull request merged**